modules/codegen/src/smithy4s/codegen/internals/Renderer.scala

```diff
@@ -801,6 +802,10 @@ private[internals] class Renderer(compilationUnit: CompilationUnit) { self =>
     )
   }
 
+  // Default toString on Throwable prints the classname, instead we use the case class toString.
+  private def defaultThrowableGetMessage =
+    line"override def toString(): $string_ = $scalaRuntime._toString(this)"
```

Comment:
Sorry for reviewing this only now. Anyway, the documentation of the ScalaRunTime object states the following : 

```scala
/** The object ScalaRunTime provides support methods required by
 *  the scala runtime.  All these methods should be considered
 *  outside the API and subject to change or removal without notice.
 */
```

therefore, I'm not comfortable with this implementation. Can you elaborate on the rationale for wanting to use this ?

Reply:
I missed that completely, that's fair.

> In any case, we can render an actual implementation that'd be using a StringBuilder quite easily.

Any format we prefer? For `Foo(bar: String)`, default implementation renders `Foo(barValue)`, we can choose how to render if we implement it ourselves